app/facades/items_facade.rb

```diff
@@ -0,0 +1,17 @@
+class ItemsFacade
+  class << self
+    def all_items
+      items = ItemService.get_data('')
+
+      items[:data].map do |item|
+        Item.new(item)
+      end
+    end
+
+    def item_by_id(id)
+      item = ItemService.get_data("#{id}")
+
+      Item.new(item[:data])
+    end
```

Comment:
no need to interpolate id, you already have it in the parameter on line 11. So, you can just pass id as is. `ItemService.get_data(id)`, and then interpolate it into the URI in your service.